VERSION

```diff
@@ -1 +1 @@
-2.0.0
+2.90.0
```

Comment:
Why 2.90. ? This would make sense when the next release would be 3.0.0.

But lets assume we release 2.1.0 wouldn't it then look somehow weird? Could we stick to the version number on master and increase it in the next release whatever number it will be?

Reply:
@elfenpiff when do you call `iox-roudi --version` how do you know it is from master or a release?

Reply:
Do we really need this distinction? But in the end `--version` should either print a version number which is consistent or just state `master`. This means when we set it here to `2.90` and release `2.1` then a earlier master version would have a higher version number than the latest release.
This would be weird.

Reply:
If we use an arbitrary number, please either use `42` or `666` :grin:

Reply:
Actually 9999 is not an arbitrary number it is somehow the highest version number and could be compared to always the latest & greatest which would be valid for master.

So when you compare the versions and want to have always the latest stuff you can compare the major number and see: 9999 > 2

Reply:
@elfenpiff well, sometimes the user just hast the binary and does not know from which sources it is compiled. In this case you would be quite happy to know the version in a bug report

Reply:
@dkroenke your decission 42, 666 or 9999 ... or maybe 99, then we could sing the song 99 red CIs

Reply:
2.90 it is.... closed